doc/ref/mloop.xml

```diff
@@ -739,14 +739,15 @@ is an example in the &GAP; code where the idea is actually used.
 <P/>
 <ManSection>
 <Func Name="Where" Arg='nr'/>
-
+<Func Name="WhereWithVars" Arg='nr'/>
 <Description>
 <Index Subkey="GAP3 name for Where">Backtrace</Index>
 <Index>Stack trace</Index>
 shows the last <A>nr</A> commands on the execution stack during whose execution
 the error occurred. If not given, <A>nr</A> defaults to 5. (Assume, for the
 following example, that after the last example <Ref Func="OnBreak"/>
-has been set back to its default value.)
+has been set back to its default value.). <Ref Func="WhereWithVars"/> also
+shows the function arguments and local variables in each stack frame
```

Comment:
```suggestion
shows the function arguments and local variables in each stack frame.
```
The term "stack frame" is not actually defined in the manual, is it? In that case, it might be better to either define it, or avoid it here.